src/main/java/mate/academy/springboot/swagger/service/SortService.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.springboot.swagger.service;
+
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public interface SortService {
```

Comment:
If this service only performs parsing but not sorting, it would be better to name it another way